src/main/scala/nl.knaw.dans.easy.bagstore/component/BagStoreComponent.scala

```diff
@@ -159,21 +159,17 @@ trait BagStoreComponent {
         for {
           bagDir <- fileSystem.toLocation(bagId)
           itemPath <- itemId.toFileId.map(f => bagDir.resolve(f.path)).orElse(Success(bagDir))
-          _ <- validateThatBagDirIsNotHidden(bagDir, itemId, forceInactive) // if the bag is hidden, also don't return items from the bag
           fileIds <- enumFiles(itemId)
           fileSpecs <- fileIds.filter(!_.isDirectory).map {
             fileId =>
               fileSystem
                 .toRealLocation(fileId)
                 .map(source => createEntrySpec(Some(source), bagDir, itemPath, fileId))
           }.collectResults
-          dirSpecs <- Try {
-            fileIds.filter(_.isDirectory).map {
-              dir =>
-                createEntrySpec(None, bagDir, itemPath, dir)
-            }
-          }
+          dirSpecs <- createDirSpecs(bagDir, itemPath, fileIds)
```

Comment:
please note that I also did this refactoring (and more) in https://github.com/DANS-KNAW/easy-bag-store/pull/81. @janvanmansum please review and merge my PR such that @stephanHogenboom can continue on it.

Reply:
I think your branch must be merged to the master asap. Then i will merge your branch in mine